specs/projectreuniondeployment/projectreuniondeployment.md

```diff
@@ -0,0 +1,138 @@
+
+# Background
+
+This is the spec for proposal [Project Reunion Deployment Information API #155](https://github.com/microsoft/ProjectReunion/issues/https://github.com/microsoft/ProjectReunion/issues/155).
+
+Project Reunion's deployment consists of a Main package and Framework package published by
```

Comment:
It would be worth describing what a "Main package" and "Framework Package" is for folks who aren't familiar with the terms. I don't even know what a Main package is 😊